docs/scheduler-perf/scheduler-perf.md

```diff
@@ -0,0 +1,856 @@
+        containerImage: quay.io/podified-antelope-centos9/openstack-cinder-scheduler:current-podified
+        customServiceConfig: |
+          [DEFAULT]
+          debug=False
+        debug:
+          service: false
+        replicas: 1
+        resources: {}
+      cinderVolumes:
+        lvm-iscsi:
+          containerImage: quay.io/podified-antelope-centos9/openstack-cinder-volume:current-podified
+          customServiceConfig: |
+            [DEFAULT]
+            debug = False
+            [lvm]
+            max_over_subscription_ratio=10000000
+            image_volume_cache_enabled=false
+            volume_driver=cinder.volume.drivers.lvm.LVMVolumeDriver
+            volume_group=cinder-volumes
+            target_protocol=iscsi
+            target_helper=lioadm
+            volume_backend_name=lvm_iscsi
+          debug:
+            service: true
+          networkAttachments:
+          - storage
+          replicas: 1
+          resources: {}
+      customServiceConfig: |
+        [DEFAULT]
+        #service_down_time=180
+        debug=True
+      databaseInstance: openstack
+      databaseUser: cinder
+      debug:
+        dbSync: false
+      passwordSelectors:
+        database: CinderDatabasePassword
+        service: CinderPassword
+      preserveJobs: false
+      rabbitMqClusterName: rabbitmq
+      secret: osp-secret
+      serviceUser: cinder
+```
+
+The cinder volume service was setup to not start automatically to allow us to
+modify the code and reduce the impact on the node of the volume creation part:
+```bash
+$ oc exec -it cinder-volume-lvm-iscsi-0 -- /bin/bash
+$ vi /usr/lib/python3.9/site-packages/cinder/volume/flows/manager/create_volume.py
+```
+And in the `_create_raw_volume` method changing:
+```python
+   ret = self.driver.create_volume(volume)
+```
+With:
+```python
+   # ret = self.driver.create_volume(volume)
+   ret = {}
+```
+
+Then inside the container the service was started:
+```bash
+$ /usr/local/bin/kolla_set_configs
+$ /usr/local/bin/kolla_start
+```
+
+We leave that terminal alone to let the cinder volume service run.
+
+From the host we now modify the quotas to remove the limit on volumes and
+gigabytes for the tests:
+```bash
+$ oc exec -it openstackclient -- openstack quota set --gigabytes=-1 --volumes=-1 admin
+```
+
+The final preparation step is to prepare the access to RabbitMQ from outside of
+OpenShift.
+
+We add a route to be able to access the RabbitMQ management web server applying
+the following manifest:
+```yaml
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    component: rabbitmq-http
+    public: "true"
+    service: rabbitmq
+  name: rabbitmq-http
+  namespace: openstack
+spec:
+  host: rabbitmq-http-openstack.apps-crc.testing
+  port:
+    targetPort: management
+  to:
+    kind: Service
+    name: rabbitmq
+    weight: 100
+  wildcardPolicy: None
+```
+
+We get the cinder credentials (<user>:<password>) for RabbmitMQ by looking at
```

Comment:
-1: This didn't render correctly, because markdown doesn't preserve the angle brackets and so they end up looking like (bogus) HTML tags. Swapping them with HTML character codes [1] worked for me.

[1] https://www.coderepublics.com/Reference/html-character-entities.php